src/org/jetbrains/plugins/scala/lang/psi/types/Compatibility.scala

```diff
@@ -143,39 +147,41 @@ object Compatibility {
     ProgressManager.checkCanceled()
     var undefSubst = new ScUndefinedSubstitutor
 
-    val clashedAssignments = clashedAssignmentsIn(exprs)
-        
-    if(clashedAssignments.nonEmpty) {
-      val problems = clashedAssignments.map(new ParameterSpecifiedMultipleTimes(_))
-      return ConformanceExtResult(problems)
-    }
-    
-    //optimization:
-    val hasRepeated = parameters.exists(_.isRepeated)
-    val maxParams: Int = if(hasRepeated) scala.Int.MaxValue else parameters.length
+    {
```

Comment:
Don't do this, remove empty line above => code will not compile, so this is bad idea generally.